public/index.html

```diff
@@ -1,10 +1,10 @@
 <!doctype html>
 <html lang="en-us">
   <head>
-      <title>Discussions | edX</title>
+      <title>Discussions | <%= process.env.SITE_NAME %></title>
```

Comment:
Not really a comment about this PR - I see the same pattern on all frontend apps. Is it possible to translate this string?

Reply:
I don't think so! However we can switch to using react-helmet, which will allow updating this dynamically per page. That can then use translations.